### DISCUSSION TITLE
When are we going to commit these changes?

### DISCUSSION THREAD

**Discussion by Orbitalz-eth** [Mikubill/sd-webui-controlnet#712] [General]
It has been two weeks and there's no update for controlnet. the canny option is unusable.

**Comment by ljleb** [comment]
@Mikubill You are probably busy at the moment, and I do not intend to pressure you in any way or anything. In light of the relatively slow pace of progress on this repo's issues and feature requests as of recently, would you be comfortable with me merging 1 or 2 PRs and then overseeing and fixing arising unpredicted problems?

@Orbitalz-eth I agree with you, however note that no one is paid to work on this project. It's okay to build up expectations from maintainers, but Mikubill does not have to spend time on this project if they are unable to do that. The code is open, you can merge any of the current PRs locally if you really need some of the proposed code changes. Again, not that I disagree with you. It's just that we cannot expect anyone to work on these projects consistently as there is no monetary incentive or legal liability.
